### PR TITLE
wallet-ext: auto-lock accounts based on user activity

### PR DESCRIPTION
**Pull request by pchrysochoidis**
## Description 

* enables auto-lock accounts setting
* users can select the time lock the wallet after  specific inactivity time
* note wallet can be locked before that when the session storage is cleared from the browser
* inactivity is based on mouse movement and pressing keys on keyboard
* an alarm is set up when an account is unlocked or user changes the auto-lock time, to trigger after that time and uses user activity to postpone it

https://github.com/MystenLabs/sui/assets/10210143/900654ee-1c95-4ad3-82ae-09e810cbe67e

https://github.com/MystenLabs/sui/assets/10210143/1db1d688-6abe-4fa9-bc20-50c3ce4b3333

closes APPS-1605

## Test Plan 

👀 

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Comment by mystie711** [comment]
Noticing that the height of input field and the dropdown is selector different,
image

**Comment by pchrysochoidis** [comment]
> Noticing that the height of input field and the dropdown is selector different, image

Thanks Abhi, I have a bug for this select box [here](https://mysten.atlassian.net/browse/APPS-1693). I will update it with that ticket.

**Review by mystie711 — APPROVED** [review]

File: apps/wallet/src/background/Window.ts [hunk 1]
```diff
@@ -4,7 +4,8 @@
 import { filter, fromEventPattern, share, take, takeWhile } from 'rxjs';
 import Browser from 'webextension-polyfill';
 
-import { POPUP_HEIGHT, POPUP_WIDTH } from '_src/ui/app/wallet/constants';
+const POPUP_WIDTH = 360;
+const POPUP_HEIGHT = 600;
```
Comment (Jibz-Mysten) [inline apps/wallet/src/background/Window.ts:8]: why remove this `import { POPUP_HEIGHT, POPUP_WIDTH } from '_src/ui/app/wallet/constants';`

**Review by Jibz-Mysten — APPROVED** [review]
LGTM

File: apps/wallet/src/background/Window.ts (hunk 1 above)
Comment (pchrysochoidis) [reply apps/wallet/src/background/Window.ts:8]: it was only used here and I wanted to delete the wallet/constants file

**Pull request merged**